ログイン画面周りのステート管理をvuexに置き換えた

shiratoriren opened the pull request:
現状ログイン処理の際のリダイレクトで
Top.vueに記述してある、画面ロード時に一部の画面表示に使用しているステートを初期状態にする処理が実行されて
ログイン処理の成功失敗関わらずログイン画面が非表示になってしまう

shiratoriren committed:
add:vuexにログイン画面を表示するかどうかのステートを追加

shiratoriren committed:
refact:ログイン画面に使うステートを整理、変更した

loginState→visibleLoginBtn
×ボタンクリック時にトップで画面でログイン画面をオーバーレイ表示するためのステートをfalseにする処理を追加
不要なstyleの削除

shiratoriren committed:
change:ログイン画面をオーバーレイ表示するための記述を追記

ヘッダーのログインボタンを押すとログイン画面表示用のステートをtrueに
トップ画面にログイン画面がオーバーレイで表示される

shiratoriren committed:
delete:使用しないログインチェックを削除

shiratoriren committed:
add:ログインロード画面の作成

shiratoriren committed:
refact:ログイン画面のロゴの読み込みが遅い現象を解決

Yudai-Saito commented on frontend/src/components/Account/Login.vue:
```diff
@@ -42,53 +32,51 @@
-  signInWithRedirect,
   onAuthStateChanged,
-  getIdToken,
+  //getIdToken,
```
使用していないので削除します！

Yudai-Saito commented on frontend/src/router/index.js:
```diff
@@ -21,6 +21,16 @@ const routes = [
+    name: "Login",
+    // route level code-splitting
+    // this generates a separate chunk (about.[hash].js) for this route
+    // which is lazy-loaded when the route is visited.
```
ここのコメントって必要？忘れないうちに消しておいてもいいかも！

Yudai-Saito committed:
refacotr: 使用していないインポートを削除

Yudai-Saito committed:
refactor: 不必要なコメントの削除とパス指定を＠からに変更

意味のないコメントの削除

componentだけアロー関数でインポートしてるのが違和感あるので直接インポート

@は/srcから始まるという意味を持っているそう

Yudai-Saito commented:
@shiratoriren 
コードはLGTMです！
